conn/channel.go

```diff
@@ -898,23 +853,11 @@ func (hc *channelSession) processTopicMessage(msg *channelMessage) {
 	hc.topicMu.RLock()
 	handler, ok := hc.topicHandlers[topic.topic]
 	hc.topicMu.RUnlock()
-	if ok && strings.Contains(topic.topic, authChannelPrefix) {
-		jsonStruct := struct {
-			Context []byte
-			MsgSeq  string
-		}{
-			topic.data,
-			msg.uuid,
-		}
-		jsonBytes, err := json.Marshal(jsonStruct)
-		if err != nil {
-			// fmt.Printf("json marshal failed, err: %v\n", jsonBytes)
-			return
-		}
-		go handler(jsonBytes)
-		return
+	responseData := new([]byte)
+	if ok {
+		handler(topic.data, responseData)
```

Comment:
the parameter of *[]byte is only used to process random number signatures? How to use by user?